src/inject/core/appDetails.js

```diff
@@ -14,4 +14,14 @@ window.addEventListener('load', () => {
       },
     });
   }
+  if (window.$) {
+    const modal = $('#about');
+    $('[data-app-version]').text(remote.app.getVersion());
```

Comment:
If we are adding this line can we delete line 7.  It would result in duplicate processing